content/docs/introduction/billing.md

```diff
@@ -37,8 +37,9 @@ See [Billing and usage metrics explained](#billing-and-usage-metrics-explained)
 **Notes:**
 
 - The limits described above are plan defaults. If you want to adjust the limits to tailor a plan to your specific requirements, please contact [Sales](https://neon.tech/contact-sales).
-- The Pro plan has a 200 GB safety limit on storage to protect your account from unintended usage but supports any data size. To increase the safety limit for your Pro account, simply submit a [Sales](https://neon.tech/contact-sales) request with the data storage size that you require.
-- The Free Tier has a 100 hour _compute active time per month_ limit, but you are always able to connect to the compute endpoint assigned to the primary branch of your Neon project, which ensures that access to data on the primary branch of your project is never interrupted.
+- The Pro plan has a 200 GB safety limit on storage to protect your account from misuse but supports any data size. To increase the safety limit for your Pro account, simply submit a [Sales](https://neon.tech/contact-sales) request with the data storage size that you require.
```

Comment:
'Safety' is internal reasoning, I'd prefer to don't reference it thins way in the docs. And rather call 'default' with the same suggestion about increasing it via support request

Reply:
Replaced with "default"